src/integrationtests/java/util/IntegrationTestBase.java

```diff
@@ -44,42 +46,61 @@
 
 public abstract class IntegrationTestBase extends AbstractTestExecutionListener {
 
-    private static final Logger LOGGER = LoggerFactory.getLogger(IntegrationTestBase.class);
+    protected RabbitTemplate rabbitTemplate;
     protected static final String MAILHOG_DATABASE_NAME = "mailhog";
-    private static final String EIFFEL_DATABASE_NAME = "eiffel";
-    private static final String EIFFEL_INTELLIGENCE_DATABASE_NAME = "eiffel_intelligence";
-
     @Autowired
-    private RmqHandler rmqHandler;
-
-    @Autowired
-    public ObjectHandler objectHandler;
+    protected MongoDBHandler mongoDBHandler;
 
-    @Autowired
-    private RulesHandler rulesHandler;
+    private static final Logger LOGGER = LoggerFactory.getLogger(IntegrationTestBase.class);
+    private static final String EIFFEL_INTELLIGENCE_DATABASE_NAME = "eiffel_intelligence";
 
     @Autowired
-    private MongoDBHandler mongoDBHandler;
+    public ObjectHandler objectHandler;
 
     @Value("${spring.data.mongodb.database}")
     private String database;
-
     @Value("${event_object_map.collection.name}")
```

Comment:
this is also fetched with System.getProperty. Use only one

Reply:
They get null if i use System.getProperty. I can make the other ones using System.getProperty to use "@Value" instead. I guess the spring properties aren't set as system property but system properties seems to be set to spring properties